packages/animation/src/parts/emptyAnimationPart.tsx

```diff
@@ -17,24 +17,29 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
-import { __, _x } from '@googleforcreators/i18n';
+import type { PropsWithChildren, HTMLAttributes } from 'react';
 
 /**
  * Internal dependencies
  */
-import { FIELD_TYPES } from '../../constants';
-import { AnimationInputPropTypes } from '../types';
+import type { AnimationPart } from './types';
 
-export const DropEffectInputPropTypes = {
-  duration: PropTypes.shape(AnimationInputPropTypes),
-};
+function EmptyAMPTarget({
+  children,
+  ...rest
+}: PropsWithChildren<HTMLAttributes<HTMLDivElement>>) {
+  return <div {...rest}>{children}</div>;
+}
 
-export default {
-  duration: {
-    label: __('Duration', 'web-stories'),
-    type: FIELD_TYPES.NUMBER,
-    unit: _x('ms', 'Time in milliseconds', 'web-stories'),
-    defaultValue: 1600,
-  },
-};
+function emptyAnimationPart(): AnimationPart {
+  return {
+    id: '-1',
```

Comment:
Is it `-1` because the type is `string`?

Reply:
Note even sure why I did that. I think it was just to be able to see that it worked somewhere. I could be anything, as long as this whole animation part doesn't produce any keyframes.